### PR TITLE
Fix display of vertical thumbnail images into documents

### LINKED ISSUE #1303: Bug: Image thumbnails display
**Describe the bug**
Vertical images inserted into documents are often too big and blurry

**Screenshots**
Example 1 (https://www.camptocamp.org/routes/53823/fr/tete-des-fetoules-arete-w):
image

Example 2 (https://www.camptocamp.org/articles/269090/fr/les-coinceurs-protections-actives-coinceurs-mecaniques):
image

**Additional context**
Current css is:
```css
figure[c2c\:size='MI'] {
  width: 25vw;
  max-width: 400px;
  img {
    width: 25vw;
    max-width: 400px;
  }
}
```
I don't know the reasoning behind (maybe some kind of mix between not too blurry, but still be 'big enough' one the page), but IMHO, we should have the same max-width and max-height.
Current state is well for horizontal images, but bad for vertical ones

### PR DESCRIPTION
**Pull request by lbesson**

### CODE REVIEW

File: src/components/generics/Markdown.vue [hunk 1]
```diff
@@ -222,12 +222,16 @@ figure[c2c\:role='embedded-figure'] {
 
 // medium image
 figure[c2c\:size='MI'] {
-  width: 25vw;
   max-width: 400px;
```
Comment (brunobesson) [inline src/components/generics/Markdown.vue:225]: only IE11 in our current targeted browsers do not support `min()`. This compatibility layer should be removed when support is dropped. I added the mention in #1131

**Comment by brunobesson** [comment]
Please rename your PR with a meaningfull title as this will be used for generating the changelog.

**Comment by lbesson** [comment]
Isn't it the issue's title which is used?

**Comment by brunobesson** [comment]
Indeed, I remove my comment as soon as I published it, but the mail was sent anyway.

**Pull request merged**